pkg/util/testing/wrappers.go

```diff
@@ -30,15 +30,16 @@ type JobSetWrapper struct {
 }
 
 // MakeJobSet creates a wrapper for a JobSet.
-func MakeJobSet(name, ns string) *JobSetWrapper {
+func MakeJobSet(name, ns string, suspend bool) *JobSetWrapper {
```

Comment:
add the SetSuspend func instead to continue with the wrapper pattern

Reply:
done.